src/dnx.coreclr/dnx.coreclr.cpp

```diff
@@ -242,8 +243,30 @@ HRESULT ExecuteMain(ICLRRuntimeHost2* pCLRRuntimeHost, PCALL_APPLICATION_MAIN_DA
         return hr;
     }
 
+    auto windows_version = dnx::utils::get_windows_version();
+    if (!windows_version)
+    {
+        trace_writer.write(L"Unexpected windows version", false);
```

Comment:
Possibly. My understanding this can only happen if someone runs pre-win7. My understanding is that post win10 will return win10 (or win8?) until we update the manifest.